interfaces/Output_from_core.atd

```diff
@@ -181,20 +181,26 @@ type skip_reason = [
 
 type time = {
   targets: target_time list;
-  ?rule_parse_time: float option; (* not used in spacegrep *)
+  rules: string list;
```

Comment:
why do we return the list of rules here? Maybe a comment explaining why in the .atd?